synapse_data_warehouse/synapse/procs/R2.13.0__add_list_downloaders.sql

```diff
@@ -0,0 +1,41 @@
+CREATE OR REPLACE PROCEDURE list_downloaders(start_record_date string, entity_list string)
+RETURNS TABLE (user_id integer, user_name varchar, email varchar, synapse_profile varchar, num_downloads integer, earliest_download_time timestamp, latest_download_time timestamp)
+LANGUAGE SQL
+AS
+declare
+    rs resultset;
+    query_str1 varchar default (
+    'with recursive filetree (parent_id, id, node_type) as (
+    select null as parent_id, id, node_type
+    from node_latest
+    where id in (' || :entity_list || ')
+    union all
+    select fc.parent_id, fc.id, fc.node_type
+    from node_latest fc
+    join filetree ft on ft.id = fc.parent_id   
+    ),');
+    query_str2 varchar default    
+    'download (user_id, timestamp, entity_id) as (
+    select fd.user_id, fd.timestamp, fd.association_object_id as entity_id
+    from filedownload fd
+    join filetree ft on ft.id = fd.association_object_id
```

Comment:
`node_latest` only contains the entities that have a Synapse ID, however, files stored within a table that have a column type of `File` don't have Synapse IDs. These only have the ID of the file handle, and that file handle ID is attached to the table through a `filehandleassociation` table [(See this related Jira ticket)](https://sagebionetworks.jira.com/browse/PLFM-8336)

Suppose I wanted to get download statistics for one or more of these kinds of files, should this be supported in this stored procedure?

Phrased another way: 
Given a table in Synapse that contains a column of type `File`, suppose I want to retrieve the stats on the downloads for one or more rows in that table. Is there (Or should there be) a stored procedure for this functionality?